### DISCUSSION TITLE
Origin Unreachable Host Error

### DISCUSSION THREAD

**Discussion by murdockma** [community/community#50245] [Other Feature Feedback, Questions, & Ideas] [answered]
### Github Pages

Hello, I'm trying to deploy a github page, and am getting an origin unreachable status (error code 523). Does anyone know what might be going on here? 

Updated

**Comment by naimur-reza** [comment, marked as answer]
There is everything ok!

**Reply by murdockma** [reply]
oh ok, it was acting up on my end. Thanks for testing it on yours.
